examples/nfct_test.rs

```diff
@@ -0,0 +1,170 @@
+                        Err(TockError::Command(CommandError {
+                            return_code: -4, /* EOFF: Not Ready */
+                            ..
+                        })) => (),
+                        Err(TockError::Command(CommandError {
+                            return_code: value, ..
+                        })) => writeln!(console, " -- Err({})!", value).unwrap(),
+                        Err(_) => writeln!(console, " -- RX Err").unwrap(),
+                    }
+
+                    match rx_buf[0] {
+                        0xe0 /* RATS */=> {
+                            let mut answer_to_select = [0x05, 0x78, 0x80, 0xB1, 0x00];
+                            bench_transmit(&mut console, &timer, "TX: ATS", &mut answer_to_select);
+                        }
+                        0xc2 /* DESELECT */ => {
+                            // Ignore the request
+                            let mut command_error = [0x6A, 0x81];
+                            bench_transmit(&mut console, &timer, "TX: DESELECT", &mut command_error);
+                        }
+                        0x02 | 0x03 /* APDU Prefix */ => match rx_buf[2] {
+                            // If the received packet is applet selection command (FIDO 2)
+                            0xa4 /* SELECT */ => if rx_buf[3] == 0x04 && rx_buf[5] == 0x08 && rx_buf[6] == 0xa0 {
+                                    // Vesion: "FIDO_2_0"
+                                    let mut reply = [rx_buf[0], 0x46, 0x49, 0x44, 0x4f, 0x5f, 0x32, 0x5f, 0x30, 0x90, 0x00,];
+                                    bench_transmit(&mut console, &timer, "TX: Version Str", &mut reply);
+                                } else {
+                                    let mut reply = [rx_buf[0], 0x90, 0x00];
+                                    bench_transmit(&mut console, &timer, "TX: 0x9000", &mut reply);
+                                }
+                            0xb0 /* READ */ =>  match rx_buf[5] {
+                                    0x02 => {
+                                    let mut reply = [rx_buf[0], 0x12, 0x90, 0x00,];
+                                    bench_transmit(&mut console, &timer, "TX: File Size", &mut reply);
+                                }
+                                0x12 => {
+                                    let mut reply = [rx_buf[0], 0xd1, 0x01, 0x0e, 0x55, 0x77, 0x77, 0x77, 0x2e, 0x6f, 0x70, 0x65, 0x6e, 0x73, 0x6b, 0x2e, 0x64, 0x65, 0x76, 0x90, 0x00,];
+                                    bench_transmit(&mut console, &timer, "TX: NDEF", &mut reply);
+                                }
+                                0x0f => {
+                                    let mut reply = [rx_buf[0], 0x00, 0x0f, 0x20, 0x00, 0x7f, 0x00, 0x7f, 0x04, 0x06, 0xe1, 0x04, 0x00, 0x7f, 0x00, 0x00, 0x90, 0x00,];
+                                    bench_transmit(&mut console, &timer, "TX: CC", &mut reply);
```

Comment:
GitHub doesn't correctly display this line on my screen. I assume it's too long :)